### DISCUSSION TITLE
[Question] How to add new line like in the demo site?

### DISCUSSION THREAD

**Discussion by ghost** [adityatelange/hugo-PaperMod#845] [Q&A]
Hey. In the demo site there's some text which uses new lines. How do make new lines? Thanks.

![image](https://user-images.githubusercontent.com/101518438/159407034-52d17142-a504-4ae7-9e6b-4eece35469e8.png)

**Comment by DejavuMoe** [comment]
You can see how the configuration file is written for the example site by referring to below line: https://github.com/adityatelange/hugo-PaperMod/blob/b8bdf094a39614166d15b6ae7e758a4066ffed71/config.yml#L130-L137
